README.md

```diff
@@ -21,16 +21,15 @@ We have created some deployments to get you started in just a few seconds. (But
 
 ### OpenShift:
 ```bash
-oc create -fhttps://github.com/tnozicka/openshift-acme/raw/master/deploy/{deploymentconfig-letsencrypt-staging,service}.yaml
+oc create -f https://github.com/tnozicka/openshift-acme/raw/master/deploy/{clusterrole,deploymentconfig-letsencrypt-staging,service}.yaml
```

Comment:
I don't think this could work. The space was missing intentionally to ensure proper shell expansion with `-f` flag for every file in `{}` - `kubectl` needs `-f` for every resource AFAIK